<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeContextWidget</class>
 <widget class="QWidget" name="TimeContextWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>27</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="checkBegin">
     <property name="text">
      <string>Starting Time</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTimeEdit" name="timeBegin">
     <property name="calendarPopup">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkEnd">
     <property name="text">
      <string>Finishing Time</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTimeEdit" name="timeEnd">
     <property name="calendarPopup">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
